test/conformance.go

```diff
@@ -39,12 +39,17 @@ const (
 	SingleThreadedImage = "singlethreaded"
 	Timeout             = "timeout"
 	WorkingDir          = "workingdir"
+	ServingContainer    = "docker.io/savita3020/servingcontainer"
+	SidecarContainerOne = "docker.io/savita3020/sidecarcontainerone"
+	SidecarContainerTwo = "docker.io/savita3020/sidecarcontainertwo"
```

Comment:
Not sure how to add image to standard `gcr` registry like other images
ex: `helloworld`
These changes are temporary to make CI pass

Will remove once i get the information on how to make images same like existing `helloworld` image

Reply:
You just need to add them to `/test/test_images` like the others.